Sophia.md

```diff
@@ -140,96 +70,59 @@ Sophia has the following types:
 | transactions | An append only list of blockchain transactions |
 | events   | An append only list of blockchain events (or log entries) |
 
-### Let  binding and Type annotations
-#### let bindings
+### Assignment (let binding), and types 
+
+#### Let bidings
+A let binding assigns a value to a named variable. These can be seen and referenced by code following them. They have this form:
 
-A "let binding", in other languages, might be called a "variable declaration/assignment". let gives names to values. They can be seen and referenced by code that comes after them.
-##### Syntax:
-```ocaml
-let varName = expression 
 ```
-Examples:
-```ocaml
-let greeting = "hello!"
+let greeting = "Hello"
 let score = 10
 let newScore = 10 + score
 let condition = false
 ```
-##### Bindings Are Immutable
-"Immutable" as in, "doesn't change". Once a binding refers to a value, it cannot refer to anything else (unless it explicitly contains a mutable value, discussed later). However, you may create a new binding of the same name which shadows the previous binding; from that point onward, the binding will refer to the newly assigned value.
+Bindings are immutable--when you have assigned a value to a variable, you may only assign a different value to the variable if you have declared the value as mutable. However you may create a new let binding, which will replace the previous one:
```

Comment:
The only mutable thing is the contract state (which is updated using the `put` function). There's no way to declare any thing else mutable.